src/Tickets/Commerce/Gateways/Stripe/Settings.php

```diff
@@ -140,7 +141,27 @@ public function __construct() {
 	 * @since TBD
 	 */
 	private function set_connection_status() {
-		$this->connection_status = tribe( Merchant::class )->get_connection_status();
+		$this->connection_status = tribe( Merchant::class )->check_account_status();
+	}
+
+	/**
+	 * Trigger a dismissable admin notice if Tickets Commerce and Stripe currencies are not the same.
+	 *
+	 * @since TBD
+	 */
+	public function alert_currency_mismatch() {
+		$stripe_currency = strtoupper( tribe( Merchant::class )->get_merchant_currency() );
+		$site_currency = strtoupper( Currency::get_currency_code() );
+
+		if ( $site_currency === $stripe_currency ) {
+			return;
+		}
+
+		tribe( Notice_Handler::class )->trigger_admin( 'tc-stripe-currency-mismatch', [
+			'content' => esc_html__(
+				sprintf( 'Tickets Commerce is configured to use %s as its currency but the default currency for the connected stripe account is %s. If you believe this is an error, you can modify the Tickets Commerce currency in the main Payments tab. Using different currencies for Tickets Commerce and Stripe may result in exchange rates and conversions being handled by Stripe.',
+					$site_currency, $stripe_currency ), 'event-tickets' )
```

Comment:
This should be the other way round.

```suggestion
			'content' => 
				sprintf( 
					// Translators: %1$s The tickets commerce currency. %2$s: The currency from the Stripe account.
					__( 'Tickets Commerce is configured to use %1$s as its currency but the default currency for the connected stripe account is %2$s. If you believe this is an error, you can modify the Tickets Commerce currency in the main Payments tab. Using different currencies for Tickets Commerce and Stripe may result in exchange rates and conversions being handled by Stripe.', 'event-tickets' )
					$site_currency, $stripe_currency ),
```